pyproject.toml

```diff
@@ -0,0 +1,5 @@
+[build-system]
+requires = ["setuptools>=44", "wheel", "setuptools_scm[toml]>=3.4.3"]
```

Comment:
I am not sure we want to switch to `setuptools_scm` just now. Let's have a ticket and do it later, and keep a manual crafted version for now.

Reply:
Actually this makes sense. Let's do it.